### PR TITLE
cmd/operator-sdk: OLM integration `alpha run/cleanup` CLI

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:** 'olm up/down' commands will deploy and delete an operator using OLM via manifests stored in an in-cluster registry-server managed by the SDK.

* cmd/operator-sdk/alpha/olm: add operator up/down commands

**Motivation for the change:** `operator-sdk` should provide tools to interact with OLM. See #1912 for past discussion of CLI.

Closes #1912.

### CODE REVIEW

File: doc/cli/operator-sdk_alpha_run.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Run an Operator in a variety of environments
+
+```
+operator-sdk alpha run [flags]
+```
+
+### Options
+
+```
+      --force-registry            Force deletion of the in-cluster registry. This option is a no-op on 'up'.
```
Comment (joelanford) [inline doc/cli/operator-sdk_alpha_run.md]: Curious about this: `This option is a no-op on 'up'.`

Does `up` need to be `run`? But that would make it seem like it's always a no-op. I'm a little confused...

File: internal/olm/operator/operator.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ const (
 type OLMCmd struct { // nolint:golint
 	// ManifestsDir is a directory containing a package manifest and N bundles
 	// of the operator's CSV and CRD's. OperatorVersion can be set to the
-	// version of the desired operator version's subdir and Up()/Down() will
+	// version of the desired operator version's subdir and Run()/Down() will
```
Comment (joelanford) [inline internal/olm/operator/operator.go]:
```suggestion
	// version of the desired operator version's subdir and Run()/Cleanup() will
```

File: doc/cli/operator-sdk_alpha_run.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  -h, --help                      help for run
+      --include strings           Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by up/down
```
Comment (joelanford) [inline doc/cli/operator-sdk_alpha_run.md]:
```suggestion
      --include strings           Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
```

File: doc/cli/operator-sdk_alpha_run.md (hunk 1 above)
Comment (estroz) [reply doc/cli/operator-sdk_alpha_run.md]: This flag should be specific to `cleanup` so I've moved it to `cleanup.NewCmd()`.

**Pull request merged**